cns/middlewares/k8sSwiftV2.go

```diff
@@ -203,7 +203,7 @@ func (k *K8sSWIFTv2Middleware) getIPConfig(ctx context.Context, podInfo cns.PodI
 				err        error
 			)
 			switch {
-			case interfaceInfo.DeviceType == v1alpha1.DeviceTypeVnetNIC && !interfaceInfo.AccelnetEnabled:
+			case interfaceInfo.DeviceType == v1alpha1.DeviceTypeVnetNIC:
 				nicType = cns.DelegatedVMNIC
 			case interfaceInfo.DeviceType == v1alpha1.DeviceTypeVnetNIC && interfaceInfo.AccelnetEnabled:
```

Comment:
we should just remove this "case interfaceInfo.DeviceType == v1alpha1.DeviceTypeVnetNIC && interfaceInfo.AccelnetEnabled" condition, and bring it back when releasing the accelnetenabled support for linux.